nntrainer/layers/cl_layers/concat_cl.h

```diff
@@ -129,13 +132,11 @@ class ConcatLayerCl : public Layer {
    * @param[in] input1_height   represents the height of the input tensor
    * @param[in] input1_width   represents the width of the input tensor A
    * @param[in] input2_width   represents the width of the input tensor X
-   * @param[in] context RunLayerContext reference
    */
   void concat_cl_axis3(const float *matAdata, const float *vecXdata,
                        float *vecYdata, unsigned int input1_batch_size,
                        unsigned int input1_channels, unsigned int input1_height,
-                       unsigned int input1_width, unsigned int input2_width,
-                       RunLayerContext &context);
+                       unsigned int input1_width, unsigned int input2_width);
 
```

Comment:
Please add `ENABLE_FP16` condition to this header.

```suggestion
#ifdef ENABLE_FP16
```
Besides this function prototypes, all `concat_cl_axis*_fp16` should be imported only when `ENABLE_FP16` is defined.
Please consider this conditional compilation for `cpp` file as well.